packages/components/src/card/test/index.js

```diff
@@ -1,78 +1,99 @@
-import { shallow } from 'enzyme';
+/**
+ * @jest-environment jsdom
+ */
+
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
 import Card from '../';
 import CompactCard from '../compact';
 
 describe( 'Card', () => {
 	// it should have a class of `card`
 	test( 'should have `card` class', () => {
```

Comment:
I noticed this test doesn't technically check if the `card` class exists anymore. I guess we could rename it or add back the class check and remove the snapshot?

Reply:
I've renamed the test. Checking the classname makes even less sense to me because it will always be there. That's a good territory for snapshot tests, so leaving that.